faithcomesbyhearing/dbp
# FCBH-2696 add congrats screen after key request

# Description

* Create views for dbp key requested

## Issue Link


[FCBH-2696](https://fullstacklabs.atlassian.net/browse/FCBH-2696)

## How Do I QA This
* visit {dbp-local}/key/requested
* use the form, check the modal of the terms

**Note:** To run the test suite refer to the Readme.md
